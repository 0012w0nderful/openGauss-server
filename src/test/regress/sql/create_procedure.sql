create procedure test_procedure_test(int,int)
SHIPPABLE IMMUTABLE
as
begin
	select $1 + $2;
end;
/

create or replace function test2(space boolean default true) return integer as
declare result integer;
begin
if(space is null) then
perform oracle."put$json_printer.pretty_print".test1(12) into result;
return result;
else
return oracle."put$json_printer.pretty_print".test1(15);
end if;
end;
/

drop function test2;
drop procedure test_procedure_test;
