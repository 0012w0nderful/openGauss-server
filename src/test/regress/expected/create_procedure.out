create procedure test_procedure_test(int,int)
SHIPPABLE IMMUTABLE
as
begin
	select $1 + $2;
end;
/

create or replace function test2(space boolean default true) return integer as
declare result integer;
begin
if(space is null) then
perform oracle."put$json_printer.pretty_print".test1(12) into result;
return result;
else
return oracle."put$json_printer.pretty_print".test1(15);
end if;
end;
/
ERROR:  schema "oracle" does not exist
CONTEXT:  compilation of PL/pgSQL function "test2" near line 2

drop function test2;
ERROR:  function test2 does not exist
drop procedure test_procedure_test;
